app/gui/src/controller/searcher/component/builder.rs

```diff
@@ -185,6 +186,9 @@ impl List {
         for (id, entry) in ids_and_entries {
             self.allowed_favorites.insert(id);
             let component = Component::new_from_database_entry(id, entry.clone_ref());
+            if component_is_private(&component) {
```

Comment:
I know this was not part of this task, but we should display private components after key-stroke. Can we add `ctrl + option + p` here? Just like `ctrl + option + <number>` changes debug view.

Reply:
The flag readable by controllers could be stored in the IDE controller, because it's accessible from the searcher controller.

Anyway, I would prefer to do it in a separate PR (not necessarily separate task: tasks can be done in more than one PR.

Reply:
I'll open a new PR for this addition.